project/templates/configs/springboard.ini

```diff
@@ -0,0 +1,97 @@
+[app:springboardfrontend]
+use = egg:springboard_{{ app_type }}
+unicore.content_repos = unicore_frontend_{{app_type}}_{{country}}
+
+pyramid.default_locale_name = {{default_language}}
+
+featured_languages =
+    {{default_language}}
```

Comment:
Ok cool. Can you open an issue on springboard so that we implement the fallback?